ivy/functional/frontends/numpy/mathematical_functions/rounding.py

```diff
@@ -113,7 +113,12 @@ def _rint(
 @handle_numpy_out
 @to_ivy_arrays_and_back
 @from_zero_dim_arrays_to_scalar
-def around(a, decimals=0, out=None):
+def _around(
+    a,
+    decimals=0,
+    out=None,
+):
     if ivy.shape(a) == ():
         a = ivy.expand_dims(a, axis=0)
```

Comment:
why is this necessary?

Reply:
It was to handle the potential event of a zero dimensional that would have been an input 'a' and change it to 1d. This was done by the person who previously tried to implement this function.